resources/views/livewire/datatables/header-no-hide.blade.php

```diff
@@ -1,22 +1,39 @@
 @unless($column['hidden'])
-<div class="relative table-cell h-12 overflow-hidden align-top" @if (isset($column['width']))style="width:{{ $column['width'] }}"@endif>
-    @if($column['unsortable'])
-        <div class="w-full h-full px-6 py-3 border-b border-gray-200 bg-gray-50 text-left text-xs leading-4 font-medium text-gray-500 uppercase tracking-wider flex items-center focus:outline-none @if($column['align'] === 'right') justify-end @elseif($column['align'] === 'center') justify-center @endif">
-            <span class="inline ">{{ str_replace('_', ' ', $column['label']) }}</span>
-        </div>
-    @else
-        <button wire:click="sort('{{ $index }}')" class="w-full h-full px-6 py-3 border-b border-gray-200 bg-gray-50 text-left text-xs leading-4 font-medium text-gray-500 uppercase tracking-wider flex items-center focus:outline-none @if($column['align'] === 'right') justify-end @elseif($column['align'] === 'center') justify-center @endif">
-            <span class="inline ">{{ str_replace('_', ' ', $column['label']) }}</span>
-            <span class="inline text-xs text-blue-400">
-                @if($sort === $index)
-                    @if($direction)
-                        <x-icons.chevron-up wire:loading.remove class="h-6 w-6 text-green-600 stroke-current" />
+    <div class="relative table-cell h-12 overflow-hidden align-top"
+         @if (isset($column['width']))style="width:{{ $column['width'] }}"@endif>
+        @if($column['unsortable'])
+            <div
+                class="w-full h-full px-6 py-3 border-b border-gray-200 bg-gray-50 text-left text-xs leading-4 font-medium text-gray-500 uppercase tracking-wider flex items-center focus:outline-none @if($column['align'] === 'right') justify-end @elseif($column['align'] === 'center') justify-center @endif">
+                <span class="inline ">{{ str_replace('_', ' ', $column['label']) }}</span>
+            </div>
+        @else
+            <button wire:click="sort('{{ $index }}')"
+                    class="w-full h-full px-6 py-3 border-b border-gray-200 bg-gray-50 text-left text-xs leading-4 font-medium text-gray-500 uppercase tracking-wider flex items-center focus:outline-none @if($column['align'] === 'right') justify-end @elseif($column['align'] === 'center') justify-center @endif">
```

Comment:
I've just looked at it and there is a question more than a review actually, 
Is that okay to have this long  class attribute

In my opinion, it reduces readability so [@apply](https://tailwindcss.com/docs/functions-and-directives#apply) would be handy in this situation ( and I lean toward that solution)
on the other hand, it seems that it's going to be used one time so it may be redundant to create custom CSS for just one time.
So yeah that's just my thought about it and you may tell me what's better from your perspective

Reply:
I think all of your points make sense. + In tailwind, you can also create components for classes that will be reused in multiple places. This code is the default from livewire-datatables, we haven't edited or added this. The difference is the code format.

Reply:
Yeah, I figured. But I thought since you're refactoring you may also consider this 
and I wanted to know your approach for such a scenario so I could learn